src/main/java/org/springframework/data/couchbase/core/NonReactiveSupportWrapper.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
This class isn't necessary.  template.support() of the Reactive*OperationSupport classes will do the same thing.